07LessonSeven/homework/css/style.css

```diff
@@ -13,10 +13,11 @@ h1 {
   color:white;
   background-color: #363636;
   width:auto;
-  padding-right: 20px;
-  height:80px;
-  padding-top: 30px;
+  padding-right: 15px;
+  height:70px;
+  padding-top: 25px;
   box-sizing: border-box;
+  font-size: 14px;
 }
 
```

Comment:
great work on this project! it's really close to being centered on my screen, but if you want to use % instead of px on your widths, it could be flexible to the screen size.